collector/info_schema_processlist_test.go

```diff
@@ -0,0 +1,77 @@
+// Copyright 2018 The Prometheus Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package collector
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/DATA-DOG/go-sqlmock"
+	"github.com/go-kit/log"
+	"github.com/prometheus/client_golang/prometheus"
+	dto "github.com/prometheus/client_model/go"
+	"github.com/smartystreets/goconvey/convey"
+)
+
+func TestScrapeProcesslist(t *testing.T) {
```

Comment:
In order for this to work, you need to parse the command line flags.

Something like this:
```go
  _, err := kingpin.CommandLine.Parse([]string{"--collect.info_schema.processlist", ""})
  if err != nil {
    t.Fatal(err)
  }
```

Reply:
Cool, I was trying to find how to pass this but didn't have a luck. Thanks!